its/ruling/src/test/resources/autoscan/autoscan-diff-by-rules.json

```
@@ -2867,4 +2867,4 @@
     "falseNegatives": 32,
     "falsePositives": 0
   }
-]
\ No newline at end of file
+]
```

No need to change the file
```suggestion
]

```

---

Yes, my mistake!